packages/admin/cms-admin/src/dam/blocks/DamFileDownloadLinkBlock.tsx

```diff
@@ -0,0 +1,134 @@
+                    openFileType: DamFileDownloadLinkBlockData["openFileType"];
+                    file?: DamFileDownloadLinkBlockData["file"];
+                }>
+                    onSubmit={(newValues) => {
+                        updateState({
+                            file: newValues.file ?? undefined,
+                            openFileType: newValues.openFileType,
+                        });
+                    }}
+                    initialValues={{
+                        file: state.file,
+                        openFileType: state.openFileType ?? "Download",
+                    }}
+                >
+                    {state.file === undefined ? (
+                        <Field name="file" component={FileField} fullWidth />
+                    ) : (
+                        <AdminComponentPaper disablePadding>
+                            <Box padding={3}>
+                                <Typography variant="subtitle1">{state.file.name}</Typography>
+                                <Typography variant="body1" color="textSecondary">
+                                    <DamPathLazy fileId={state.file.id} />
+                                </Typography>
+                            </Box>
+                            <Divider />
+                            <AdminComponentButton startIcon={<Delete />} onClick={() => updateState({ ...state, file: undefined })}>
+                                <FormattedMessage {...messages.empty} />
+                            </AdminComponentButton>
+                        </AdminComponentPaper>
+                    )}
+
+                    <Divider />
+                    <AdminComponentPaper>
+                        <Field
+                            name="openFileType"
+                            fullWidth
+                            label={<FormattedMessage id="blocks.damFileDownloadLink.openFileType" defaultMessage="Open file" />}
+                        >
+                            {(props) => (
+                                <>
+                                    <FinalFormSelect {...props}>
+                                        <MenuItem value="Download">
+                                            <FormattedMessage id="blocks.damFileDownloadLink.openFileType.download" defaultMessage="as a download" />
+                                        </MenuItem>
+                                        <MenuItem value="NewTab">
+                                            <FormattedMessage id="blocks.damFileDownloadLink.openFileType.newTab" defaultMessage="in a new tab" />
+                                        </MenuItem>
+                                    </FinalFormSelect>
+                                </>
+                            )}
+                        </Field>
+                    </AdminComponentPaper>
+                </BlocksFinalForm>
+            </>
+        );
+    },
+
+    isValid: (state) => {
+        return !!state.file;
```

Comment:
This prevents saving the block with default values, which we always want to support: https://docs.comet-dxp.com/docs/blocks/block-best-practices#allow-saving-a-block-with-its-default-values